rust-toolchain.toml

```diff
@@ -1,2 +0,0 @@
-[toolchain]
-channel = "nightly"
```

Comment:
Thank you for your response. Yes I fixed both requests but I think that I need some help to write an appropriate test.

Reply:
You can add files to `tests/rust` where your toml file has the new option.

Reply:
Ok, in my newest commit I added a test crate package_version that checks whether the file is generated with the correct package version. 
Since some other tests failed with cython, I modified cbindgen. Now Comments such as header, trailer, etc. are written to the file after a check if the comment should be in C/C++ syntax (/*) or Python syntax ('''). 

`cargo test` now tells `ok. 145 passed;`